libraries/networking/src/UserActivityLogger.h

```diff
@@ -55,6 +59,7 @@ private slots:
 private:
     UserActivityLogger();
     Setting::Handle<bool> _disabled { "UserActivityLoggerDisabled", true };
```

Comment:
Should we change this line as well while we're at it?

```suggestion
    Setting::Handle<bool> _disabled { "UserActivityLoggerDisabled", false };
```

Reply:
That would effectively enable the activity logger by default, I'd say hold off on that for now.

Reply:
So we'll leave it as "apparently enabled" but not really working unless you uncheck and check it?

...I think I'm okay with that actually.

Reply:
Our wording gets confusing really fast since the variable is for a "disabled" state, so when we say enabled or disabled... I get confused about which we're referring to lol.

Reply:
It would not be accurate to not say that you are not unconfused.